packages/wallet/src/redux/protocols/transaction-submission/components/wait-for-confirmation.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { Fragment } from 'react';
+
+interface Props {
+  name: string;
+  transactionId: string;
+  networkId: number;
+}
+
+export default class WaitForXConfirmation extends React.PureComponent<Props> {
+  buildEtherscanLink() {
+    switch (this.props.networkId) {
+      case 1:
```

Comment:
@tomclose This might be cleaner with an Enum

Reply:
Yeah - that's worth looking at but probably as future work. In this case, I just did a copy-and-paste from the code that was already in the `src/components` folder ...